java/core/src/java/org/apache/orc/impl/WriterImpl.java

```diff
@@ -182,6 +184,8 @@ public WriterImpl(FileSystem fs,
 
     this.encodingStrategy = opts.getEncodingStrategy();
     this.compressionStrategy = opts.getCompressionStrategy();
+    this.compressionZstdLevel = opts.getCompressionZstdLevel();
+    this.compressionZstdWindowLog = opts.getCompressionZstdWindowLog();
```

Comment:
Is it possible to work with compressionStrategy? For example, compressionStrategy indicates we need to look for external zstd options. BTW, are they actually used?

Reply:
Thanks. Removed.